docs/quickstart.md

```diff
@@ -0,0 +1,766 @@
+#### Remote test
+You may also deploy your application into AWS Account by issuing the following command.
+```bash
+> sam build && sam deploy --guided
+...
+CloudFormation outputs from deployed stack
+------------------------------------------------------------------------------------------------------------------------------------------
+Outputs                                                                                                                                  
+------------------------------------------------------------------------------------------------------------------------------------------
+Key                 HelloWorldFunctionIamRole                                                                                            
+Description         Implicit IAM Role created for Hello World function                                                                   
+Value               arn:aws:iam::123456789012:role/sam-app-HelloWorldFunctionRole-1T2W3H9LZHGGV                                          
+
+Key                 HelloWorldApi                                                                                                        
+Description         API Gateway endpoint URL for Prod stage for Hello World function                                                     
+Value               https://1234567890.execute-api.eu-central-1.amazonaws.com/Prod/hello/                                                
+
+Key                 HelloWorldFunction                                                                                                   
+Description         Hello World Lambda Function ARN                                                                                      
+Value               arn:aws:lambda:eu-central-1:123456789012:function:sam-app-HelloWorldFunction-dOcfAtYoEiGo                           
+------------------------------------------------------------------------------------------------------------------------------------------
+Successfully created/updated stack - sam-app in eu-central-1
+```
+This command builds a package and deploy it to your AWS Account. You find the API Url against which you can launch requests in the output section. Now, you can trigger your endpoints.
+```bash
+> curl https://1234567890.execute-api.eu-central-1.amazonaws.com/Prod/hello
+{"statusCode":200,"body":{"message":"hello unknown!"}}%
+```
+!!! Info
+    For more details on the SAM deployment mechanism, see [link](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-deploy.html).
+
+## API Gateway router
+Let's say we want to have different method that acts like an echo server. It takes user input (username) and output it to the caller. We need to create an API with an URL path `/hello/<name>`, where the `name` string is the input from the user.
+One approach is to create another lambda with required method and set up the API gateway to call it.
```

Comment:
Not sure I understood what you mean here -- Consider reviewing the wording. Also, use capital `L` for Lambda

Reply:
Rewrote to this:
```
Let's expand our application with a new method. It takes  an username as a input and return it in the response.

We decided to write another Lambda including required method. Next, we configure our API Gateway to expose this Lambda under new unique path `/hello/{name}`.
```